### PR TITLE
Port docs over to Sphinx

### PR DESCRIPTION
**Pull request by petroav**
Once I merge this I'll delete the current README, create a github-pages branch, commit the built docs there and point to them from the new README.

### CODE REVIEW

**Comment by petroav** [comment]
@mattsfuller can you review when you get a chance?

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python
```
Comment (anusudarsan) [inline docs/conf.py]: these files should be having copyright?

Comment (petroav) [inline docs/conf.py]: Yep, I'll add the license header.

**Comment by anusudarsan** [comment]
looks good mod comment. Also the travis build is failing. Did we have a green build before for this project?

**Comment by petroav** [comment]
Thanks for the review. I'll push in a bit after adding the licenses. There was never a Travis job setup for this project. I'll file a ticket to set one up.

**Pull request closed without merging**